source/symmetry/imap/grammar.d

```diff
@@ -6,8 +6,9 @@ module symmetry.imap.grammar;
     a PEG grammer.
 +/
 
-version (SIL) :
-    import pegged.grammar;
+version (Have_pegged):
```

Comment:
This unused module was previously enabled for the SIL plugin and `sil-library` config. Not anymore - there's the `with-grammar` config now instead (build tested by CI).